modules/product/src/main/java/com/opengamma/strata/product/fxopt/FxSingleBarrierOptionTrade.java

```diff
@@ -66,7 +67,7 @@
   * The premium sign should be compatible with the product Long/Short flag.
   * This means that the premium is negative for long and positive for short.
   */
-  @PropertyDefinition(validate = "notNull")
+  @PropertyDefinition(get = "optional")
```

Comment:
@jodastephen would need your review especially here.
I think it is the change needed.